src/group/libp2p_group_relcast_server.erl

```diff
@@ -357,7 +357,7 @@ handle_info(force_close, State=#state{}) ->
     %% down the group by exiting the supervisor.
     spawn(fun() ->
                   lager:info("removing group for force_close timeout"),
-                  libp2p_swarm:remove_group(State#state.tid, State#state.group_id)
+                  libp2p_group_mgr:remove_group(State#state.tid, State#state.group_id)
```

Comment:
Why is this needed, given that libp2p_swarm:remove_group calls libp2p_group_mgr:remove_group?

Reply:
Actually, this calls via the TID, but libp2p_group_mgr:remove_group expects a pid as the first argument?